modulus/utils/capture.py

```diff
@@ -348,6 +357,8 @@ class StaticCaptureTraining(_StaticCapture):
         Number of warmup steps for cuda graphs, by default 11
     amp_type : Union[float16, bfloat16], optional
         Auto casting type for AMP, by default torch.float16
+    gradient_clip_norm : Union[float, None], optional
+        Threshold for gradient clipping
     label : Optional[str, None], optional
```

Comment:
Nit: probably meant `Optional[str]`, no?

Reply:
Good point, I standardize the rest of the docstrings to match.